.github/workflows/release.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on new tags
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        name: Check out code
+      - name: Create Draft Release
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OWNER: ${{ github.repository_owner }}
+          REPO: ${{ github.event.repository.name }}
+        run: |
+          GH_EXTRA_ARGS=""
```

Comment:
This does not work as expected. See https://github.com/NVIDIA/nvidia-container-toolkit/actions/runs/9549456867/job/26319106625

Also when running the gh command line as given, it asks a number of questions. Not sure whether that would be skipped in the action.

Reply:
Fixed with `github.ref_name`

Reply:
It should not ask questions, as it should know is running on GItHub Actions mode https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows